src/components/formatSchoolName/formatSchoolName.test.tsx

```diff
@@ -0,0 +1,21 @@
+import { render } from "@testing-library/react";
+
+import { formatSchoolName } from "./formatSchoolName";
+
+describe("formatSchoolName", () => {
+  it("should return the expected div with input value in bold", () => {
+    const { getByText, getByTestId } = render(
+      formatSchoolName(
+        "Macaulay Church of England Primary School, Lambeth, SW4 0NU",
+        "ima",
+      ),
+    );
+
+    const divWithText = getByText("ry School, Lambeth, SW4 0NU");
+    expect(divWithText).toBeInTheDocument();
+    const stronElement = getByText("ima");
+    expect(stronElement).toBeInTheDocument();
+    const strongElement = getByTestId("strong-element");
+    expect(strongElement.tagName).toBe("STRONG");
```

Comment:
might be a bit more useful to test that the text content of the strong element matches what we expect?